html/videos.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+
+<head>
+  <script src="https://code.jquery.com/jquery-3.5.1.slim.min.js" integrity="sha384-DfXdz2htPH0lsSSs5nCTpuj/zy4C+OGpamoFVy38MVBnE+IbbVYUew+OrCXaRkfj" crossorigin="anonymous"></script>
+  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.0/dist/umd/popper.min.js" integrity="sha384-Q6E9RHvbIyZFJoft+2mJbHaEWldlvI9IOYy5n3zV9zzTtmI3UksdQRVvoxMfooAo" crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+  <link rel="stylesheet" href="../css/carousel.css">
+  <link rel="stylesheet" href="../css/main.css">
+  <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/js/bootstrap.min.js" integrity="sha384-OgVRvuATP1z7JjHLkuOU7Xw704+h835Lr+6QL9UvYjZE3Ipu6Tp75j7Bh/kR0JKI" crossorigin="anonymous"></script>
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta charset="utf-8">
+  <title>Hawken Coding</title>
+</head>
+
+<body>
+  <!--Navbar-->
+  <nav class="navbar navbar-expand-md navbar-dark bg-dark">
+    <a class="navbar-brand" href="../index.html">Hawken Coding</a>
+    <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarsExample04" aria-controls="navbarsExample04" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+
+    <div class="collapse navbar-collapse" id="navbarsExample04">
+      <ul class="navbar-nav mr-auto">
+        <li class="nav-item">
+          <a class="nav-link" href="../index.html#AboutUs">About Us</a>
+        </li>
+        <li class="nav-item dropdown">
+          <a class="nav-link dropdown-toggle" href="projects.html" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            Projects
+          </a>
+          <div class="dropdown-menu" aria-labelledby="navbarDropdown">
+            <a class="dropdown-item" href="#John">John</a>
+            <a class="dropdown-item" href="#Spencer">Spencer</a>
+            <a class="dropdown-item" href="#Nikhil">Nikhil</a>
+          </div>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="../html/videos.html">Videos</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+
+  <div id="AboutUs" class="container-fluid">
+    <h1>Helpful Videos</h1>
+    <h2 class="text-center">Video #1 (this will be the title)</h2>
```

Comment:
this is a really good start. It will probably look really nice but something that would make this even better is to make the videos into bootstrap cards. This is something that bootstrap does so that you can have individual cards which can contain different things. I'll drop a link below. Also consider directly linking the video into the site using an iframe. This lets people play the video without leaving our site. Ill also put a link to that below.

Bootstrap cards: https://getbootstrap.com/docs/4.0/components/card/
iFrame for Youtube vids: https://www.w3schools.com/html/html_youtube.asp

Reply:
oh also for the bootstrap cards there's a example on the projects page if you want to figure out how the structure works. you can put whatever inside a card it just provides a nice viewpoint. I'd also check out the link though because that'll give you an idea of how to adjust the cards to your needs (e.g. how many per row, sizing one bigger than the others etc.)